tests/plugin.js

```diff
@@ -0,0 +1,35 @@
+import test from 'ava';
+import contentTypes from 'punchcard-content-types';
+import nunjucks from 'nunjucks';
+import contains from 'validator/lib/contains';
+
+import plugin from '../';
+
+contentTypes.pluginTests(test, plugin);
+
+const data = {
+  year: {
+    label: 'Please choose a year',
+    placeholder: 'year',
+    type: 'select',
+    settings: {
+      minYear: 2000,
+      maxYear: 2016,
+    },
+    value: 2006,
+  },
+};
+
+test('select fills options', t => {
+  const rendered = nunjucks.renderString(plugin.html, data);
+  t.true(contains(rendered, '<option value=\"2005\" >2005</option>'), 'Select must contain 2005');
```

Comment:
In addition to the check for dates outside the range that are inside validation?